taskipy/task.py

```diff
@@ -48,7 +53,19 @@ def __extract_task_command(self, task_toml_contents: object) -> str:
             except KeyError:
                 raise MalformedTaskError(self.__task_name, 'the task item does not have the "cmd" property')
 
-        raise MalformedTaskError(self.__task_name, 'tasks must be strings, or dicts that contain { cmd, help, use_vars }')
+        raise MalformedTaskError(self.__task_name, 'tasks must be strings, or dicts that contain { cmd, cwd, help, use_vars }')
+
+    def __extract_task_workdir(self, task_toml_contents: object) -> Optional[str]:
+        if isinstance(task_toml_contents, str):
+            return None
+
+        if isinstance(task_toml_contents, dict):
+            try:
+                return task_toml_contents['cwd']
```

Comment:
I was thinking: maybe we should also assert that `cwd` is a string, and throw if it's not. wdyt?

Reply:
I think it will be good!

Reply:
Done on eff3a19e36a72d4a7644681c0cb61dc759aa499c